vue.config.js

```diff
@@ -45,7 +45,6 @@ module.exports = {
                     icon: "./build/icons/icon.icns"
                 },
                 linux: {
-                    target: ["pacman", "deb", "rpm"],
```

Comment:
Why did you remove this?

Reply:
Fixed, made the change while on a different branch and I guess it stuck when I switched.